lib/checkother.cpp

```diff
@@ -2025,8 +2025,8 @@ void CheckOther::duplicateAssignExpressionError(const Token *tok1, const Token *
 
     reportError(toks, Severity::style, "duplicateAssignExpression", 
         "Both variables '" + tok1->str() + "' and '" + tok2->str() + "' are assigned the same expression.\n"
```

Comment:
hmm.. one way to rephraze it could be `Same expression used in consequtive assignments of x and y.` I am not sure if this is improvement. Maybe there is a better word than "consequtive".

Reply:
I updated the message to use `Same expression used in consecutive assignments of x and y.`.

> Maybe there is a better word than "consequtive".

It spelled "consecutive", and I think it sounds fine.